build.sbt

```diff
@@ -37,12 +37,20 @@ lazy val core = project
   .settings(Testing.settings:_*)
 
 lazy val cluster = project
+  .dependsOn(core)
+  .aggregate(core)
   .settings(moduleName := "kanaloa-cluster")
   .configs(Testing.Integration)
   .settings(commonSettings:_*)
   .settings(Dependencies.settings:_*)
   .settings(Format.settings:_*)
   .settings(Publish.settings:_*)
   .settings(Testing.settings:_*)
+  .settings(ClusterTests.settings:_*)
+  .settings(
+    libraryDependencies ++= Dependencies.akkaCluster
+  ).configs(MultiJvm)
 
 
+addCommandAlias("validate", ";root;clean;compile;test;integration:test;multi-jvm:test")
```

Comment:
what about 'test-all' ?  'validate' feels a bit vague here, being that 3 of the actions here are test related.